recipes/default.rb

```diff
@@ -22,15 +22,19 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 # THE SOFTWARE.
 
+node['passenger-nginx']['ruby_version'].prepend('ruby-') if node['passenger-nginx']['ruby_version'] =~ /^\d/
 
-execute "apt-get update" do
-  command "apt-get update"
-  user "root"
-end
+if platform_family?('debian')
+  execute "apt-get update" do
+    command "apt-get update"
+    user "root"
+  end
 
-# Install basic packages
-%w(git build-essential curl libcurl4-openssl-dev libpcre3 libpcre3-dev).each do |pkg|
-  apt_package pkg  
+  # Install basic packages
+  apt_package %w(git build-essential curl libcurl4-openssl-dev libpcre3 libpcre3-dev)
```

Comment:
Weird… Works for me and the Chef docs say you can pass either a string or array to apt_package: https://docs.chef.io/resource_apt_package.html <https://docs.chef.io/resource_apt_package.html>
What version of chef are you using?

V/r
Jacob

> On Aug 25, 2015, at 16:12, Leonard Teo <notifications@github.com> wrote:
> 
> In recipes/default.rb <https://github.com/ballistiq/chef-passenger-nginx/pull/2#discussion_r37920419>:
> 
> >  
> > -# Install basic packages
> > -%w(git build-essential curl libcurl4-openssl-dev libpcre3 libpcre3-dev).each do |pkg|
> > -  apt_package pkg  
> > +  # Install basic packages
> > +  apt_package %w(git build-essential curl libcurl4-openssl-dev libpcre3 libpcre3-dev)
> This doesn't work for me. apt_package expects a string, not an array.
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/ballistiq/chef-passenger-nginx/pull/2/files#r37920419>.
>